frameworks/g-cloud-9/manifests/declaration.yml

```diff
@@ -1,41 +1,45 @@
 -
-  name: Essentials
+  name: Providing suitable services
   editable: True
   description: >
     You must make this declaration to apply to G-Cloud&nbsp;9. You can come back and change your answers at any time before
-    5pm GMT, 14 December 2016.
+    5pm BST, 11 April 2017. [might want to remove]
```

Comment:
We probably at least want to remove the bit in square brackets... or leaving it in for discussion?